test/v1/controllers/RetrievePensionsChargesControllerSpec.scala

```diff
@@ -65,22 +64,19 @@ class RetrievePensionsChargesControllerSpec
   private val deleteHateoasLink =
```

Comment:
You might like to remove the hateoas links (as mocked) and use `hateoaslinks` and `hateoaslinksJson`.